packages/cli/src/lib/cmds/build.ts

```diff
@@ -142,6 +142,25 @@ class Build {
         APP_BUNDLE_BUILD_OUTPUT_FILE_NAME, APP_BUNDLE_SIGNED_FILE_NAME);
   }
 
+  /**
+   * Based on the promptResponse to update the project or not, run an update or print the relevant warning message.
+   */
+  async runUpdate(
+      promptResponse: boolean,
+      manifestFile: string,
+      noUpdateMessage: string): Promise<boolean> {
+    if (promptResponse) {
```

Comment:
nit: do you mind turning this around and making it a guard clause, so:

```
if (!promptResponse) {
  // print message
  return true;
}
```

It just means that the meat of the function is less indented.

Reply:
good call out! done :)